meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend

```diff
@@ -34,6 +38,12 @@ do_deploy_prepend_mx6ull() {
     install -m 0644 ${DEPLOY_DIR_IMAGE}/u-boot.itb ${DEPLOYDIR}/${PN}/u-boot-mfgtool.itb
 }
 
+do_deploy_prepend_imx6ullevk() {
```

Comment:
Since this is to be used with sec, shouldn't it be prepend_imx6ullevk-sec instead?